FloatingLabel.js

```diff
@@ -22,6 +22,19 @@ class FloatingLabel extends Textbox {
     };
   }
 
+  componentWillReceiveProps(next) {
+
+    // Make sure that when we have a value, we also show the label
+    if(this.props.value !== next.value && next.value) {
+      this.setState({
+        fadeAnim: new Animated.Value(1),
+      })
+    }
+
+    super.componentWillReceiveProps(next)
```

Comment:
Is this line needed?

Reply:
Yep. The parent component sets the internal state for value. We could either use this, or set the state ourself. But I thought it would be better to let the parent class manage what it is supposed to manage anyway.